styles/arch-wiki/catppuccin.user.css

```diff
@@ -194,10 +99,23 @@
+      );
+      background: url("data:image/svg+xml,@{svg}") !important;
+      background-size: 190px 40px !important;
```

Comment:
Huh why do we need this?

Reply:
You shouldn't need this because the actual website's CSS already has it so it is redundant.

Reply:
mkmk

Reply:
> You shouldn't need this because the actual website's CSS already has it so it is redundant.

1. i have no idea what "this" is
1a. if you're talking about the sizing, that's needed, otherwise it's too huge, like the burden i have theming these userstyles (jkjk)